context/package.json

```
@@ -34,7 +34,7 @@
     "uuid": "^8.3.2",
     "vega": "^5.13.0",
     "vega-lite": "^4.13.1",
-    "vitessce": "^1.1.0",
+    "vitessce": "1.1.5",
```

```suggestion
    "vitessce": "^1.1.5",
```
Unless there's a reason, npm culture is just to `^` and leave the details to package-lock.